packages/overlay/test/renderer.test.js

```diff
@@ -1,114 +1,147 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync } from '@vaadin/testing-helpers';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import '../vaadin-overlay.js';
 
 describe('renderer', () => {
-  let rendererContent, renderModel;
+  let overlay, content;
 
-  beforeEach(() => {
-    rendererContent = document.createElement('p');
-    rendererContent.textContent = 'renderer-content';
-    renderModel = { selected: true };
```

Comment:
The actual content of the `renderModel` is never checked. Renamed it to `overlayModel` and changed to be empty object. In fact, the `model` API is not used by our components, except for `vaadin-context-menu`.